### PR TITLE
Added say command, added more responses in ask command, and updated README

### PR DESCRIPTION
**Pull request by sleepybuns**
`/say` command will have kyoyobot say a message in a specified channel. Kyoyobot will also announce which user told it to say the message in the current channel. 

Added more funny responses in the `/ask` command.

README also updated to specify this repo is a discord bot.

### CODE REVIEW

File: toes/extra_toe.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import random
+from discord import app_commands as slash, Client, Interaction, TextChannel
+from util.debug import DEBUG_GUILD
+
+DEBUG = False
+
+# This command will have the bot send a message to a specified text channel 
+# bot will announce the user who initiated the command in the current channel
+@slash.command()
+async def say(interaction: Interaction, channel: TextChannel, message:str,) -> None:
```
Comment (96-LB) [inline toes/extra_toe.py:10]:
```suggestion
async def say(interaction: Interaction, channel: TextChannel, message: str) -> None:
```

File: toes/pinky.py [hunk 2]
```diff
@@ -27,19 +27,23 @@ async def ask(interaction: Interaction, question: str) -> None:
...
         'Concentrate and ask again.',
+        'Offer more feet and try again.',
         # Negative
         'Don\'t count on it.',
         'My reply is no.',
         'My sources say no.',
         'Outlook not so good.',
         'Very doubtful.',
-	'Kyoyo says no'
+	    'Kyoyo says no',
```
Comment (96-LB) [inline toes/pinky.py:45]:
```suggestion
    'Kyoyo says no', 
```

File: toes/extra_toe.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    '''Have Kyoyobot say something in a channel.'''
+    
+    text = f'{interaction.user} told me to say \"{message}\" in #{channel.name}\n'
```
Comment (96-LB) [inline toes/extra_toe.py:13]: Maybe we should make these mentions? i.e. ping the user with <@id> and the channel with <#id>

**Review by 96-LB — CHANGES_REQUESTED** [review]
Thanks for the PR! Went through and added a few suggestions

File: toes/pinky.py (hunk 2 above)
Comment (sleepybuns) [reply toes/pinky.py:45]: ohhhh interesting, there was a tab there rather than spaces. i'll fix it.

File: toes/extra_toe.py (hunk 1 above)
Comment (sleepybuns) [reply toes/extra_toe.py:10]: oh my bad, I will fix that

File: toes/extra_toe.py (hunk 3 above)
Comment (sleepybuns) [reply toes/extra_toe.py]: Good idea, I added the mentions.

**Comment by sleepybuns** [comment]
I committed all the suggestions 👍. Let me know if I overlooked something.

**Review by 96-LB — APPROVED** [review]

**Comment by 96-LB** [comment]
Sweet, looks good! Will do some local testing tomorrow to make sure nothing's wrong.

**Pull request merged**